src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -1031,17 +1031,6 @@ module Pipe = struct
       Counter.v "transition_frontier_primary_transitions" ~help ~namespace
         ~subsystem
 
-    let router_transition_frontier_controller : Counter.t =
```

Comment:
Do we care these metrics?

Reply:
We don't. Previously we had two places where a transaction could've been dropped and one unused metric. Now we'll have just one place where transactions can be dropped, and we're removing two metrics that became unused (while one that was unused before became used 😆 )